apps/web/modules/api/v2/management/surveys/[surveyId]/contact-links/segments/[segmentId]/lib/utils.ts

```diff
@@ -0,0 +1,36 @@
+export const sanitizeFields = (fields?: string): string[] | [] => {
```

Comment:
this return type seems a bit confusing, its an union between a string array and a tuple, can't we just use the `string[]` type here for the return type of this function?